addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/serializer/CustomTankerkoenigDetailResultDeserializer.java

```diff
@@ -43,13 +43,15 @@ public TankerkoenigDetailResult deserialize(final JsonElement json, final Type t
             final Double e10 = jsonStation.get("e10").getAsDouble();
             final Double e5 = jsonStation.get("e5").getAsDouble();
             final Double diesel = jsonStation.get("diesel").getAsDouble();
+            final Boolean isOpen = jsonStation.get("isOpen").getAsBoolean();
```

Comment:
I'm only using it in this line:
`updateState(CHANNEL_STATION_OPEN, (station.isOpen() ? OpenClosedType.OPEN : OpenClosedType.CLOSED));`
IMHO that way a null would be treated the same as "false".

BTW:
This is really hard to understand! There are two ways! During the "normal" price polling only the String .status is set by the webrequest, hence I do need set the .open via code. In case of the "Detail" request (when initializing a station and after that every 24 houres) the .open is set directly from the webrequest. It took me a while as well (maybe my holiday-trip was a bit toooo long!).